### ISSUE TITLE
Allow reading timeseries inputs from a database via pandas

### ISSUE THREAD

**Issue by sjpfenninger** [calliope-project/calliope#296]
Pandas already provides a wrapper for SQL databases: https://pandas.pydata.org/pandas-docs/stable/user_guide/io.html#sql-queries

**Comment by brynpickering** [comment]
@ahilbers I wonder if this could be added to your PR (#305)?

**Comment by ahilbers** [comment]
@brynpickering Good idea -- I'll see if I can work this in as well.

**Comment by ahilbers** [comment]
Hey guys, I might be able to build this in. However, I've got a question about what you guys prefer. One way of doing this now is to do the following:
```
df = pd.read_sql_table(...)
model = calliope.Model(.., timeseries_dataframes={'ts': df})
```
where I use the method implemented in #305. This would only work inside a python script/shell and not from a command line.

Is the functionality you guys want the ability do this directly (i.e. specify something like 'sql=db_name:table_name' in the `yaml` files?). And, would this be possible to use from a command line, or just inside python? I don't have a lot of experience with using `pandas` and `SQL` together, so I can't say right now whether it would be possible to do this in a command line run.

**Comment by sjpfenninger** [comment]
In fact, since we can now use dataframes directly, we should just use that if somebody does need direct access to an SQL database. Better not to add complexity we do not need. I'll close this issue. Thanks @ahilbers anyway :-)
